src/common/helper.js

```diff
@@ -1139,16 +1139,41 @@ async function getMembersByHandles(handles) {
  * @returns {Object}
  */
 async function getStandSkills(ids) {
```

Comment:
Does this have an error handling from where it is called? If not, you need to handle the error to rollback the entire transaction if any of the batches are unsucessful in returning skill data

Reply:
It's called only by [validateSkills](https://github.com/topcoder-platform/challenge-api/blob/dev/src/common/challenge-helper.js#L107) to validate skill ids before the actual update